org.gimp.GIMP.json

```diff
@@ -0,0 +1,76 @@
+{
+    "id": "org.gimp.GIMP",
+    "branch": "stable",
+    "base": "org.gimp.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "gimp-2.8",
+    "rename-desktop-file": "gimp.desktop",
+    "rename-appdata-file": "gimp.appdata.xml",
+    "rename-icon": "gimp",
+    "finish-args": ["--share=ipc", "--socket=x11", "--filesystem=host",
+                    "--filesystem=xdg-config/GIMP", "--share=network",
+                    "--socket=system-bus", "--socket=session-bus" ],
+    "tags": ["stable"],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": ["/include", "/lib/pkgconfig", "/share/pkgconfig",
+                "/share/aclocal", "/man", "/share/man", "/share/gtk-doc",
+                "/share/vala", "*.la", "*.a", "/bin/wmf*", "/bin/libwmf-*",
+                "/bin/pygtk*", "/bin/pygobject*"],
+    "modules": [
+        {
+            "name": "babl",
+            "config-opts": [ "--disable-docs" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/babl",
+                    "branch": "BABL_0_1_10",
+                    "commit": "0aa5c964e466e87dd4842863d63dfc989a3ef4d3"
+                }
+            ]
+        },
+        {
+            "name": "gegl",
+            "config-opts": [ "--disable-docs", "--disable-introspection" ],
+            "cleanup": [ "/bin" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/gegl",
+                    "branch": "GEGL_0_2_0",
+                    "commit": "57e23d310291a632ea8e16b7b8e5b97f7f289680"
+                }
+            ]
+        },
+        {
+            "name": "gimp",
+            "config-opts": [ "--disable-docs", "--disable-gtk-doc", "--disable-gtk-doc-html", "--enable-vector-icons" ],
+            "cleanup": [ "/bin/gimptool-2.0", "/bin/gimp-console-2.9" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/gimp",
+                    "branch": "GIMP_2_8_22",
+                    "commit": "e39a4e12039a255f29d7cf79365190f01c5f37aa"
+                }
+            ],
+	    "post-install": [
```

Comment:
These should just be in `cleanup`.

Reply:
Indeed. Cf https://github.com/flatpak/flatpak/issues/1082 for the bug about cleanup not properly working when cleaning the BaseApp files from the main manifest.